test/test_app_details.py

```diff
@@ -31,15 +31,21 @@ def env_app_root():
     yield
     del os.environ['APP_ROOT']
 
+@pytest.fixture(scope='function')
+def env_qradar_app_id():
+    os.environ['QRADAR_APP_ID'] = '1005'
+    yield
+    del os.environ['QRADAR_APP_ID']
+
 # ==== get_app_id ====
 
 @patch(GET_MANIFEST_JSON, return_value=manifest_path('installed.json'))
```

Comment:
patch not required as we're no longer using the manifest. Removing the patch lets you remove mock_manifest argument too.